src/test/ui/macros/auxiliary/issue-84429.rs

```diff
@@ -0,0 +1,4 @@
+#[macro_export]
```

Comment:
You may want to explicitly set the edition to `2018` to be sure that the macro will always be compile in this edition.
```suggestion
// edition:2018

#[macro_export]
```

Reply:
Well I think it's compiled in 2015 edition by default and unlikely to change, just like invoking `rustc` without the `--edition` flag. I have no preference here, though.